tools/webdriver/webdriver/client.py

```diff
@@ -425,7 +429,26 @@ def send_command(self, method, url, body=None):
             an error.
         """
         response = self.transport.send(method, url, body)
-        value = response.body["value"]
+
+        if "value" in response.body:
+            value = response.body["value"]
+        elif "error" in response.body:
+            logger.warning("'value' key not found in response but 'error' key "
+                           "found in root. This is a compliance issue, but the tests are "
+                           "working around it.")
+            value = response.body
```

Comment:
Not sure I agree with what you say in the commit message:

> ```
> Chrome currently doesn't use anything much like the spec
> protocol. Nevertheless it's useful to run higher level tests without
> failing just because we can't create a session or similar.
> ```

Why do we want to work around this instead of just failing the tests? Logging that the implementation is incorrect but not actually failing the test gives a false impression that the implementation is conformant.